test/wikidata_ids_decorator_test.rb

```diff
@@ -1,21 +1,51 @@
 require 'test_helper'
 
 describe WikidataIdsDecorator do
-  it 'has a version number' do
-    ::WikidataIdsDecorator::VERSION.wont_be_nil
+  describe 'gem' do
+    it 'has a version number' do
+      ::WikidataIdsDecorator::VERSION.wont_be_nil
+    end
   end
+end
+
+describe WikidataIdsDecorator::Links do
+  around { |test| VCR.use_cassette(File.basename(url), &test) }
+  let(:response) { Scraped::Request.new(url: url).response }
+  let(:decorator) { WikidataIdsDecorator::Links.new(response: response) }
 
   describe 'Wikipedia page with links' do
+    let(:url) { 'https://en.wikipedia.org/wiki/Douglas_Adams_(disambiguation)' }
```

Comment:
I don't really follow why you're including this page, to test against, if all you're testing is that you get a Q42 link. If that link is also available from '/wiki/Life,_the_Universe_and_Everything', why not make it one of the tests against that? (which would also let you simplify things more as every section would be against the same `:url`)